src/frontend/utils/initAnonymousId.ts

```diff
@@ -0,0 +1,15 @@
+import { getDecodedJwt } from 'data/appData';
+import { checkLtiToken } from './checkLtiToken';
+import { getAnonymousId, setAnonymousId } from './localstorage';
+
+export const initAnonymousId = () => {
```

Comment:
IMO, what the function is doing could be more explicit if it was called `getOrInitAnonymousId`

Reply:
Done